app/main/lib/shared_models/audio_model.py

```diff
@@ -100,7 +89,8 @@ def delete(self, task):
 
     def add(self, task):
         try:
-            audio = Audio.from_url(task.get("url"), task.get("doc_id"), task.get("context", {}))
+            body = task.get("body", {})
+            audio = Audio.from_url(body.get("url"), body.get("raw", {}).get("doc_id"), body.get("context", {}), body.get("response", {}).get("hash_value"))
```

Comment:
What happens when `hash_value` is `None` here and `url` and `doc_id` are not already in the DB? 
* Do we throw an error?
* Does it continue and run a search that returns nothing?
* Does it continue and run a search that returns everything?

Reply:
I believe this would continue and run a search that returns nothing - how would you like to handle @computermacgyver ?

Reply:
I think we can leave as is. I want to test manually when I'm running the code.